x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/analytics_list/analytics_list.tsx

```diff
@@ -232,7 +248,7 @@ export const DataFrameAnalyticsList: FC<Props> = ({
   const columns = getColumns(
     expandedRowItemIds,
     setExpandedRowItemIds,
-    isManagementTable,
+    allowCreate,
```

Comment:
I am seeing the Start / Delete actions in the Management page, whereas they should be showing when inside the ML UI. Does this `allowCreate` flag need inverting somewhere?

Reply:
Good catch, boolean refactoring... I reverted to be aligned with anomaly detection jobs. 
Check https://github.com/elastic/kibana/pull/49464/commits/a9cac46a14f007118d6baa93bd0048ea21caeac6